prod/westeurope/internal/api/apim/api_management_api_io_payportal/swagger.json.tmpl

```diff
@@ -0,0 +1,236 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "IO pay portal API",
+    "contact": {
+      "name": "IO team",
+      "url": "https://forum.italia.it/c/progetto-io"
+    },
+    "description": "Documentation of the IO Pay Portal Function API here.\n"
+  },
+  "host": "${host}",
+  "basePath": "/api/v1",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/payment-requests/{rptId}": {
+      "x-swagger-router-controller": "PagoPAProxyController",
+      "parameters": [
+        {
+          "name": "rptId",
+          "in": "path",
+          "required": true,
+          "description": "Unique identifier for payments.",
+          "type": "string"
+        },
+        {
+          "name": "test",
+          "in": "query",
+          "description": "Use test environment of PagoPAClient",
+          "type": "boolean",
+          "required": false
+        }
+      ],
+      "get": {
+        "operationId": "getPaymentInfo",
+        "summary": "Get Payment Info",
+        "description": "Retrieve information about a payment",
+        "responses": {
+          "200": {
+            "description": "Payment information retrieved",
+            "schema": {
+              "$ref": "#/definitions/PaymentRequestsGetResponse"
+            },
+            "examples": {
+              "application/json": {
+                "importoSingoloVersamento": "200,",
```

Comment:
Sorry guys .... I might be wrong .... but it looks to me in all the other api definitions we always named functions and variables il english @balanza @gunzip any guidelines here?

Reply:
it's right, but for the moment it's equal to `io-backend` [api](https://github.com/pagopa/io-backend/blob/792eddfd51f1530d864dbe6684d50cd245ea9326/api_backend.yaml#L525)